platform/platform-paxweb-jettyconfig/src/main/java/org/codice/ddf/security/session/HashSessionIdManager.java

```diff
@@ -47,13 +47,14 @@
 
 import org.codice.ddf.platform.util.RandomNumberGenerator;
 import org.eclipse.jetty.server.session.AbstractSession;
-import org.eclipse.jetty.server.session.AbstractSessionIdManager;
 
 /**
  * Custom implementation of the {@link org.eclipse.jetty.server.SessionIdManager} that shares session
  * data between sessions in a cluster.
  */
-public class HashSessionIdManager extends AbstractSessionIdManager {
+@edu.umd.cs.findbugs.annotations.SuppressWarnings(value = "NM_SAME_SIMPLE_NAME_AS_SUPERCLASS", justification = "not changing")
+public class HashSessionIdManager extends org.eclipse.jetty.server.session.HashSessionIdManager {
```

Comment:
Sonarqube doesn't like it either. :/

Is a class rename out of the question? If not, we'll need to mark this as a non-issue for Sonarqube as well.